python/docs/source/python_wrapper/index.rst

```diff
@@ -0,0 +1,46 @@
+Python Wrapper Documentation
+============================
+
+Vowpal Wabbit is a fast machine learning library for online learning, and this is the python wrapper for the project.
+
+
+Code Documenation
+-----------------
+
+See documenation for the following modules in the package:
+
+.. toctree::
+   vowpalwabbit.pyvw
+   vowpalwabbit.sklearn
+
+Usage
+-----
+
+You can use the python wrapper directly like this:
+
+.. code-block:: python
+    from vowpalwabbit import pyvw
+    vw = pyvw.vw(quiet=True)
+    ex = vw.example('1 | a b c')
+    vw.learn(ex)
+    vw.predict(ex)
+Or you can use the included scikit-learn interface like this:
```

Comment:
It will be good, if we can show the output of both of these code snippets

Reply:
Here output is not important for the user so I left this.